R/deploymentTarget.R

```diff
@@ -1,93 +1,241 @@
 # calculate the deployment target based on the passed parameters and
 # any saved deployments that we have
-deploymentTarget <- function(recordPath = ".",
-                             appId = NULL,
-                             appName = NULL,
-                             appTitle = NULL,
-                             envVars = NULL,
-                             account = NULL,
-                             server = NULL,
-                             forceUpdate = FALSE,
-                             error_call = caller_env()) {
+deploymentTarget <- function(
+  recordPath = ".",
+  appId = NULL,
+  appName = NULL,
+  appTitle = NULL,
+  envVars = NULL,
+  account = NULL,
+  server = NULL,
+  forceUpdate = FALSE,
+  error_call = caller_env()
+) {
 
-  appDeployments <- deployments(
+  if (!is.null(appId)) {
+    return(deploymentTargetFromAppId(
```

Comment:
Then this becomes something like `findDeploymentWithAppId()`

Reply:
This became `findDeploymentTargetByAppId()`